packages/webpack/lib/plugins/stats.js

```diff
@@ -26,8 +28,13 @@ const analyzeCompilation = ({ chunks, chunkGroups }) => {
   return { entryChunks, vendorChunks, chunksByModule };
 };
 
-const extractFiles = ({ entryChunks, vendorChunks, chunksByModule }) => {
-  const gatherFiles = (result, { files }) => [...result, ...files];
+const extractFiles = (chunkData, rawPublicPath) => {
+  const publicPath = trimTrailingSlash(rawPublicPath);
+  const { entryChunks, vendorChunks, chunksByModule } = chunkData;
+  const gatherFiles = (result, { files }) => [
+    ...result,
+    ...files.map((file) => `${publicPath}/${file}`),
```

Comment:
Why no `path.join`? Would release you from the duty of trimming slashes.

```diff
- ...files.map((file) => `${publicPath}/${file}`),
+ ...files.map((file) => path.join(publicPath, file)),
```

Reply:
Unfortunately `join` is platform-specific: 
https://nodejs.org/api/path.html#path_path_join_path

We could possibly go for `path.posix.join`, but I always found that being explicit in cases like this had some merit.

Reply:
Ah, missed the fact that we're moving in a URL context here. So `url-join` was the weapon of choice here. But pulling that in seems a little over the top. So string concatenation would be fine for me.